config/webpack.prod.js

```diff
@@ -36,7 +36,7 @@ const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL;
 const FABRIC8_REALM = process.env.FABRIC8_REALM;
 const FABRIC8_RECOMMENDER_API_URL = process.env.FABRIC8_RECOMMENDER_API_URL || 'http://api-bayesian.dev.rdu2c.fabric8.io/api/v1/';
 const FABRIC8_SSO_API_URL = process.env.FABRIC8_SSO_API_URL;
-const FABRIC8_AUTH_API_URL = process.env.FABRIC8_AUTH_API_URL;
+const FABRIC8_AUTH_API_URL = process.env.FABRIC8_AUTH_API_URL || 'http://auth.prod-preview.openshift.io/api/';
```

Comment:
use `https://`

Reply:
thanks , it was a temporary commit - removed.